backend/src/app/rpc/commands/binfile.clj

```diff
@@ -336,11 +336,23 @@
       (->> (db/exec! conn [sql ids])
            (mapv #(assoc % :file-id id))))))
 
+(defn- get-file-tagged-object-thumbnails
+  [{:keys [::db/pool]} id]
+  (pu/with-open [conn (db/open pool)]
+    (let [sql (str "SELECT * FROM file_tagged_object_thumbnail WHERE file_id = ?")]
+      (->> (db/exec! conn [sql id])
+           (mapv #(dissoc % :file-id))))))
+
 (def ^:private storage-object-id-xf
   (comp
    (mapcat (juxt :media-id :thumbnail-id))
    (filter uuid?)))
 
+(def ^:private storage-object-id-whatever
```

Comment:
You don't need to declare a new transducer, you can continue use the `storage-object-id-xf` for all purposes